src/imports/Ubuntu/AddressBook/ContactEditor/ContactEditorPage.qml

```diff
@@ -169,14 +170,19 @@ Page {
 
     function focusToFirstEntry(field)
     {
-        var itemToFocus = field
-        if (field.repeater)
-            itemToFocus = field.repeater.itemAt(0)
-
-        if (itemToFocus) {
-            root.idleMakeMeVisible(itemToFocus)
-            itemToFocus.forceActiveFocus()
+        //only focus when new field requested
```

Comment:
Is this an intentional change? Then you should also describe it in your PR. Otherwise this is a kind of "drive-by" change that we want to prevent normally. In some cases you would even need to split it off into another PR. So make your PR text "on-topic" with that change :)

Reply:
Humm, in fact, it was not a visual issue since adding that special birthday field even if all the forceFocus things were unnecessarily fired . You're right, I will propose another PR for that and merge it into the click test to avoid feedbacks about it :)

Reply:
can you please merge the clickable https://github.com/ubports/address-book-app/pull/121 so that i have to avoid always cherry picking "-n" from it ?